src/ruby/pb/test/xds_client.rb

```diff
@@ -98,11 +103,30 @@ def create_stub(opts)
   )
 end
 
+class StatsPerMethod
```

Comment:
nit: we can replace:

```
def rpcs_started
  @rpcs_started
end
def result
  @result
end
```

with

```
attr_reader :rpcs_started, :result
```

at the top of the class definition

Reply:
Comment addressed. Did it in another place too.